README.md

```diff
@@ -48,6 +48,62 @@ cmake .. -D CMAKE_PREFIX_PATH=$CONDA_PREFIX -D CMAKE_INSTALL_PREFIX=$CONDA_PREFI
 make && make install
 ```
 
+## Installation within a mamba environment (wasm build instructions)
+
+First clone the repository, and move into that directory
+```bash
+git clone --depth=1 git@github.com:jupyter-xeus/xeus-r.git
+cd ./xeus-r
+```
+
+Now you'll want to create a clean mamba environment containing the tools you'll need to do a wasm build. This can be done by executing
```

Comment:
*"You'll"* sounds a bit informal, maybe we can replace it with more passive voice.

> Create a clean mamba environment containing the necessary build tools. This can ...